cmd/jujud-controller/agent/model/manifolds.go

```diff
@@ -426,10 +425,6 @@ func IAASManifolds(config ManifoldsConfig) dependency.Manifolds {
 			NewFirewallerFacade:      firewaller.NewFirewallerFacade,
 			NewRemoteRelationsFacade: firewaller.NewRemoteRelationsFacade,
 		})),
-		unitAssignerName: ifNotMigrating(unitassigner.Manifold(unitassigner.ManifoldConfig{
-			APICallerName: apiCallerName,
-			Logger:        config.LoggingContext.GetLogger("juju.worker.unitassigner"),
-		})),
```

Comment:
If you're deleting this here, should you not also delete all the manifolds and the facades?